js/memory.js

```diff
@@ -0,0 +1,142 @@
+$(document).ready(function() {
+
+	(function() {
+		'use strict';
+
+		var cardAlreadyShowing = null;
+		var totalTries = 0;
+		var $boardEl = $('#board');
+
+		/**
+		 * Card constructor
+		 *
+		 * @constructor
+		 * @param {string} color The color to make this card
+		 */
+		var Card = function(color) {
+			this.color = color;
+		};
+
+		Card.prototype.createEl = function() {
+			var thisCard = this;
+			this.cardEl = $('<div>')
+				.addClass('each-card')
+				.data('background-color', this.color)
+				.on('click', function () {
+					thisCard.showCard();
+					if (cardAlreadyShowing) {
+						thisCard.checkForMatch(thisCard);
+					} else {
+						cardAlreadyShowing = thisCard;
+					}
+				});
```

Comment:
You might want to move the click handler to be another method on the `Card`, just to keep `createEl()` short. I've even heard people go so far as to say "don't use anonymous functions for callbacks".